apps/parse-rated-paragraphs/src/app/parse-rated.ts

```diff
@@ -2,12 +2,20 @@ import * as fs from 'fs';
 import {
   ParsedArticleParagraphStandalone,
   ArticleParagraphBacksUpClaim,
+  ImpactFileList,
 } from '@foodmedicine/interfaces';
 
 const allParagraphsBasePath = './tmp/correlated-paragraphs/';
 
 function getAllJsonPaths(): string[] {
-  return fs.readdirSync(allParagraphsBasePath);
+  const path = allParagraphsBasePath + impactListFileName;
+  const impactList: ImpactFileList = JSON.parse(
+    fs.readFileSync(path).toString()
+  ) as ImpactFileList;
+  // get all the filenames from each impact item
+  return impactList
```

Comment:
nit: since you are doing two mappings and a .flat() operation, it might be helpful to leave a comment with a quick example of what an expected output would look like